pages/[date]/[room]/talks/[id].js

```diff
@@ -46,37 +75,76 @@ const StyledMain = styled.div`
   padding: 0 2em;
 `;
 
-const StyledBackRoute = styled(Link)`
-  text-decoration: none;
-  color: black;
-  width: 5em;
-  height: 3.5em;
-  padding: 1em 3em;
-  border-radius: 5px;
-  border: none;
-  background-color: #eaeaea;
-  display: flex;
-  justify-content: center;
-  align-items: center;
+const IconWrapper = styled.div`
+display: flex:
+justify-content: space-between;
+`;
```

Comment:
```suggestion
const IconWrapper = styled.div`
display: flex;
justify-content: space-between;
`;
```

Reply:
Haven't seen this tiny mistake but was really confused why it didn't work, thanks!